workflow/src/main/java/com/synopsys/integration/alert/workflow/message/NotificationReceiver.java

```diff
@@ -51,32 +54,44 @@
 
     @Autowired
     public NotificationReceiver(Gson gson, NotificationAccessor notificationAccessor, NotificationProcessor notificationProcessor, ChannelEventManager eventManager) {
-        super(gson, NotificationEvent.class);
+        super(gson, NotificationReceivedEvent.class);
         this.notificationAccessor = notificationAccessor;
         this.notificationProcessor = notificationProcessor;
         this.eventManager = eventManager;
     }
 
     @Override
-    public void handleEvent(NotificationEvent event) {
-        if (NotificationEvent.NOTIFICATION_EVENT_TYPE.equals(event.getDestination())) {
-            if (null == event.getNotificationIds() || event.getNotificationIds().isEmpty()) {
-                logger.warn("Can not process a notification event without notification Id's.");
-                return;
-            }
+    public void handleEvent(NotificationReceivedEvent event) {
+        if (NotificationReceivedEvent.NOTIFICATION_RECEIVED_EVENT_TYPE.equals(event.getDestination())) {
```

Comment:
Rather than nesting all of the logic inside of this if-statement, can we negate this condition and use the code from the else here? Then `return` to short-circuit.